paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/TransformSpecToElement.kt

```diff
@@ -58,8 +62,11 @@ internal class TransformSpecToElement @Inject constructor(
                 is SaveForFutureUseSpec -> it.transform(initialValues)
                 is SectionSpec -> it.transform(initialValues)
                 is MandateTextSpec -> it.transform(initialValues.merchantName)
+                // TODO combine these into a single text header spec.
                 is AfterpayClearpayTextSpec ->
                     it.transform(requireNotNull(initialValues.amount))
+                is SimpleTextHeaderSpec ->
```

Comment:
is this related to Klarna?

Reply:
fyi This was removed and we're now using a modified version of the mandate text.